cfme/tests/infrastructure/test_tenant_quota.py

```diff
@@ -253,7 +253,7 @@ def test_setting_child_quota_more_than_parent(tenants_setup, parent_quota, child
 
 
 @pytest.mark.long_running
-@pytest.mark.provider([VMwareProvider, RHEVMProvider], override=True, scope="module",
+@pytest.mark.provider([VMwareProvider], override=True, scope="module",
```

Comment:
Request: we don't use scope="module" for providers, please remove

Reply:
@rlbabyuk 
To fix this issue. If I removed **scope='module'** and changed it to **scope='function'** then I need some changes in following files too:
```
1. cfme/tests/infrastructure/test_tenant_quota.py:95:  def small_vm(provider, small_template_modscope) 
2. cfme/fixtures/provider.py:357:  def small_template_modscope(provider) 
```
In above files: need to change **scope='function'**. 
If it is not affecting other tests then we can go ahead with these changes.
please let me know if you have any suggestions for me.

Reply:
@ganeshhubale I think in this case we can leave provider scope as module, not to change other fixtures